### PR TITLE
Add summary on the files

### PR DESCRIPTION
**Pull request by lucaslovato**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,14 +28,17 @@ Andromeda allow:
...
+  * [System Bootstrap](./docs/how_to_setup.md#System-Bootstrap)
+* [Running via Docker container](./docs/how_to_run_docker_container.md)
+  * [Prerequisites](./docs/how_to_run_docker_container.md#Prerequisites)
+  * [Configuring the appsettings.json](./docs/how_to_run_docker_container.md#Configuring-the-appsettings.json)
+  * [Running the container](./docs/how_to_run_docker_container.md#Running-the-container)
+* [Exporting database into files](./docs/export_csv_json.md)
+* [Contribute to Andromeda](./docs/how_to_contribute.md)
+  * [Code of conduct](./docs/how_to_contribute.md#Code-of-conduct)
+  * [Your First Pull Request](./docs/how_to_contribute.md#You-First-Pull-Request)
+  * [How to send a Pull Request](./docs/how_to_contribute.md#How-to-send-a-Pull-Request)
```
Comment (leonardo-brickabode) [inline README.md]: I think that it is good for the Contribution part to be in a separate topic.

File: README.md [hunk 2]
```diff
@@ -28,14 +28,17 @@ Andromeda allow:
 
 The idea on making Andromeda possible came by [Jellyfish Insights](http://jellyfishinsights.com/)
```
Comment (leonardo-brickabode) [inline README.md]: Maybe we should change this to:
Andromeda was developed as part of the [Jellyfish Insights](http://jellyfishinsights.com/) and supported by the Foundation for Economic Education (FEE).

File: README.md (hunk 1 above)
Comment (leonardo-brickabode) [reply README.md]: Maybe something like this
## Contributing to Andromeda
We would love to have you contributing to the Andromeda project. You can see the instruction on how to become an Andromeda contributor [here](./docs/how_to_contribute.md).

File: README.md [hunk 3]
```diff
@@ -28,14 +28,17 @@ Andromeda allow:
...
+# Documentation
+
+* [Getting started](./docs/how_to_setup.md)
+  * [System Requirements](./docs/how_to_setup.md#System-Requirements)
+  * [System Bootstrap](./docs/how_to_setup.md#System-Bootstrap)
+* [Running via Docker container](./docs/how_to_run_docker_container.md)
+  * [Prerequisites](./docs/how_to_run_docker_container.md#Prerequisites)
+  * [Configuring the appsettings.json](./docs/how_to_run_docker_container.md#Configuring-the-appsettings.json)
+  * [Running the container](./docs/how_to_run_docker_container.md#Running-the-container)
+* [Exporting database into files](./docs/export_csv_json.md)
```
Comment (leonardo-brickabode) [inline README.md]: Maybe:
Exporting data from the data lake into files.

Comment (leonardo-brickabode) [inline README.md]: Maybe:
Exporting data from the data lake into files.

Comment (lucaslovato) [reply README.md]: Idk, *data* from the *data*.
Exporting data lake into files maybe?

File: README.md (hunk 2 above)
Comment (lucaslovato) [reply README.md]: I suggest we add BA here too.

Comment (lucaslovato) [reply README.md]: I suggest we add BA here too.

File: README.md (hunk 3 above)
Comment (leonardo-brickabode) [reply README.md]: > Exporting data lake into files maybe?
This sounds to me as if we were serializing the Database itself.
We are exporting some part of the data from the database. So maybe:
Exporting Data lake data to files.

File: README.md (hunk 1 above)
Comment (lucaslovato) [reply README.md]: Can be.

**Pull request merged**